automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -262,12 +262,12 @@ trait AzureBilling extends FixtureAnyFreeSpecLike {
   // Note that the final 'optional' field for a pre-created landing zone is not technically optional
   // If you fail to include a landing zone, the wb-libs call to create the billing project will fail, timing out due to landing zone creation not being an expected part of creation
   // Contact the workspaces team if this fails to work
-  implicit val azureManagedAppCoordinates: AzureManagedAppCoordinates = AzureManagedAppCoordinates(
-    UUID.fromString("fad90753-2022-4456-9b0a-c7e5b934e408"),
-    UUID.fromString("f557c728-871d-408c-a28b-eb6b2141a087"),
-    "staticTestingMrg",
-    Some(UUID.fromString("f41c1a97-179b-4a18-9615-5214d79ba600"))
-  )
+  // implicit val azureManagedAppCoordinates: AzureManagedAppCoordinates = AzureManagedAppCoordinates(
```

Comment:
I would feel comfortable removing the old code instead of commenting out :) What do you think?

Reply:
Yes, I'll do that after @lucymcnatt has a chance to review everything. I also need to remove references to `iv-upgrade-workflow-dispatch-to-support-run-name` in the `.yml` workflow once the other PR has been reviewed and merged. I'll leave this thread `unresolved` for now.

Reply:
oh yeah I was about to comment about that branch name, so it'll be changed back to main before merging?

Reply:
Branch name has been changed back to `main`.